src/Microsoft.ML/Models/OneVersusAll.cs

```diff
@@ -13,10 +13,15 @@ namespace Microsoft.ML.Models
     public sealed partial class OneVersusAll
     {
         /// <summary>
-        /// Create OneVersusAll multiclass trainer.
+        /// One versus all learner (also in known as One vs the rest) is a multiclass learner with
+        /// strategy to fit one classifier per class. It picks each class and trains provided binary learner
+        /// against all the other classes. 
+        /// See <a href="https://en.wikipedia.org/wiki/Multiclass_classification"/> (One-vs.-rest) section
```

Comment:
><a href="https://en.wikipedia.org/wiki/Multiclass_classification"/> [](start = 16, length = 67)

Is this a thing? Under normal circumstances I'd think that HTML would have to be formatted:

```html
<a href="[http://foo.com](http://foo.com)">Bar</a>
```

as opposed to

```html
<a href="[http://foo.com](http://foo.com)"/> Bar
```

As far as I know XML comment link anchor tags have to follow the same convention. Also note that you could have made the `#One-vs.-rest` on the URL to make it actually just go directly to that section, rather than having to instruct the user.

I might put that link on the first time you mention it in this document.

Look at me, giving advice on writing HTML. Does that mean I count as a full stack dev? :smile: